File: LLM-Translations/GPT-4/avatar/Java/Go/codeforces_59_A.go
package main

import (
	"fmt"
	"strings"
	"unicode"
)

func main() {
	var word string
	fmt.Scan(&word)

	uppercase := 0
	lowercase := 0

	for _, ch := range word {
		if unicode.IsUpper(ch) {
			uppercase++
		} else {
			lowercase++
		}
	}

	if uppercase > lowercase {
		fmt.Println(strings.ToUpper(word))
	} else {
		fmt.Println(strings.ToLower(word))
	}
}

// //End of Code

File: LLM-Translations/GPT-4/avatar/Python/C/codeforces_59_A.c
#include <stdio.h>
#include <ctype.h>
#include <string.h>

int main() {
    char txt[100];
    int cu = 0, cl = 0;
    
    printf("Enter a string: ");
    fgets(txt, sizeof(txt), stdin);
    
    for(int i = 0; txt[i]; i++) {
        if(islower(txt[i]))
            cl++;
        else if(isupper(txt[i]))
            cu++;
    }
    
    if(cu > cl) {
        for(int i = 0; txt[i]; i++) {
            txt[i] = toupper(txt[i]);
        }
    } else {
        for(int i = 0; txt[i]; i++) {
            txt[i] = tolower(txt[i]);
        }
    }
    
    printf("%s", txt);
    return 0;
} // End of Code

